fenix/app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -274,6 +274,10 @@ private fun WallpaperThumbnailItem(
         Modifier
     }
 
+    // Used to debounce the click events for setting wallpaper
+    @Suppress("MagicNumber")
+    val debounceInterval = 1000L
```

Comment:
If we use a default parameter as suggested above then we can potentially remove this magic number as well

Reply:
removed this magic number